Packages/Tracking/Physical Hands/Runtime/Scripts/UI/PhysicalHandsButton.cs

```diff
@@ -375,6 +403,15 @@ protected virtual void OnHandContactPO(ContactHand hand)
 
             if (hand != null)
             {
+                if (_contactedHands.ContainsKey(hand))
```

Comment:
same comment about primary hover - perhaps we use 2 bools, to avoid using containskey